src/Marten/Events/Projections/Aggregator.cs

```diff
@@ -9,7 +9,7 @@ namespace Marten.Events.Projections
 {
     public class Aggregator<T> : IAggregator where T : class, new()
     {
-        public static readonly string ApplyMethod = "Apply";
```

Comment:
I'd rather we don't do that. I'm purposely using `static readonly` instead of `const`